dataplane/worker.cpp

```diff
@@ -3998,6 +3993,69 @@ inline void cWorker::balancer_tunnel(rte_mbuf* mbuf,
 	preparePacket(mbuf);
 }
 
+/// Sets the IPv6 source address for the packet, taking into account the address set for the service.
+/// In order to optimize the distribution of packets across the queues of the NIC installed on the servers
+/// to which the packet will be sent, the source address is (if possible) randomized taking into account the mask.
+inline void cWorker::set_ipv6_source(rte_ipv6_hdr* header,
+                                     const ipv6_address_t& balancer_src,
+                                     const dataplane::globalBase::ipv6_prefix_t& service_src,
+                                     const rte_ipv4_hdr* ipv4HeaderInner,
+                                     const rte_ipv6_hdr* ipv6HeaderInner)
```

Comment:
`set_ipv6_source()` - is not good name.
This is need only for balancer? If not, why one of argument is `balancer_src`?